packages/doc/src/components/Home/Title.jsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const Title = styled.h1`
+  font-size: 48px;
+  font-weight: 300;
+  line-height: 64px;
+  margin: 0 0 24px;
+
+  @media (max-width: 900px) {
+    font-size: 20px;
+    line-height: 30px;
+    margin-bottom: 0;
+  }
+`;
```

Comment:
I think that we can adjust this component following the CSS Ordering, what do you think?

Reply:
Done!